src/js/serial.js

```diff
@@ -187,6 +189,28 @@ const serial = {
             }
         });
     },
+    connectVirtual: function (callback) {
+        var self = this;
+        self.openRequested = true;
+        self.connectionType = 'virtual';
+        self.logHead = 'VIRTUAL: ';
```

Comment:
Can remove `logHead` as it is replaced by using `connectionType` in console.log messages and other logic.